blueprints/dotnet/src/APIGatewayAuthorizerHandler/Error/UnauthorizedException.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace APIGatewayAuthorizerHandler.Error
+{
+    internal class UnauthorizedException : System.Exception
+    {
+        public UnauthorizedException() : base("Unauthorized")
```

Comment:
Assuming you have tested that this results in a 401 response? API Gateway matches on the `errorMessage` provided by Lambda runtime.

Reply:
@jackrk Yep, This is the reason for having a parameterless constructor which always passes "Unauthorized" as the exception message. Any other exception message won't trigger the expected 401.